### PR TITLE
Improved status update handling for mobile apps

### PR DESCRIPTION
**Pull request by nickvergessen**

### CODE REVIEW

File: lib/Controller/ChatController.php [hunk 1]
```diff
@@ -255,18 +255,19 @@ public function sendMessage(string $message, string $actorDisplayName = '', stri
...
 	 *         'actorDisplayName', 'timestamp' (in seconds and UTC timezone) and
 	 *         'message'.
 	 */
-	public function receiveMessages(int $lookIntoFuture, int $limit = 100, int $lastKnownMessageId = 0, int $timeout = 30, int $setReadMarker = 1, int $includeLastKnown = 0): DataResponse {
+	public function receiveMessages(int $lookIntoFuture, int $limit = 100, int $lastKnownMessageId = 0, int $timeout = 30, int $setReadMarker = 1, int $includeLastKnown = 0, int $noStatusUpdate = 0): DataResponse {
 		$limit = min(200, $limit);
 		$timeout = min(30, $timeout);
 
-		if ($this->participant->getSessionId() !== '0') {
+		if ($noStatusUpdate === 0 && $this->participant->getSessionId() !== '0') {
```
Comment (danxuliu) [inline lib/Controller/ChatController.php:270]: Checking the parameter here would prevent not only user status updates but also pings. I do not know if that is intended or not :-)

Comment (nickvergessen) [reply lib/Controller/ChatController.php:270]: it is

**Review by Ivansss — APPROVED** [review]
Tested and works 👍

**Pull request merged**